pkg/discovery/globalnet/globalnet_test.go

```diff
@@ -0,0 +1,99 @@
+/*
+© 2020 Red Hat, Inc. and others.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package globalnet
+
+import (
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+)
+
+var _ = Describe("IsOverlappingCidr", func() {
```

Comment:
Are these tests not run as part of CI?
I do not see them in the CI output - https://travis-ci.com/submariner-io/submariner-operator/builds/149776758

Reply:
https://travis-ci.com/submariner-io/submariner-operator/builds/149776758#L1220